x-pack/solutions/security/plugins/security_solution/common/experimental_features.ts

```diff
@@ -274,6 +274,12 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables advanced mode for Trusted Apps creation and update
    */
   trustedAppsAdvancedMode: false,
+
+  /**
+   * Enables Trusted Devices artifact management for device control protections.
+   * Allows users to manage trusted USB and external devices
+   */
+  trustedDevicesEnabled: false,
```

Comment:
Just fyi, with the work @semd did a few months ago (see [here](https://github.com/elastic/kibana/pull/217363)) you don't really need to name the feature flag `myFeatureFlagDisabled`. You can just name it `myFeatureFlag` (in your case `trustedDevices`), because now you can enable or disable it via config.